### DISCUSSION TITLE
Crop Price Guide

### DISCUSSION THREAD

**Discussion by jmmullins** [jeremyckahn/farmhand#444] [Q&A] [answered]
One of the achievement rewards is the Crop Price Guide.. where can the player see this?

**Comment by lstebner** [comment, marked as answer]
I think the word "guide" is a little misleading here. I believe the reward is this little caret next to prices indicating whether the sell value is a "good" or "bad" opportunity. it has a tooltip on hover as well.

![image](https://github.com/jeremyckahn/farmhand/assets/628757/9b8563b6-e9f4-4158-899e-b6487353e6a8)

**Reply by jmmullins** [reply]
Ah, I see! That is incredibly subtle 🙈 

I'll be submitting a more formalized request with different versions of what it could look like, but I'd love if we could expand upon the price guide idea!

**Reply by lstebner** [reply]
yeah, very subtle. i forget it's even there... would be great to hear some ideas for what the guide could be!

**Reply by jeremyckahn** [reply]
Now that you mention it, it might be worth revisiting the semantics of this achievement. I think the mechanics are good, but to @jmmullins' point it falsely implies there would be some sort of centralized UI displaying the prices.

Perhaps we could rename "Crop Price Guide" to be "Crop Price Indicators?"
